CMakeLists.txt

```diff
@@ -27,123 +30,135 @@ if (ENABLE_DEVELOPER_DEFAULTS)
   set(ENABLE_TOOLS On CACHE BOOL "")
 endif ()
 
-set(ENABLE_CUDA Off CACHE BOOL "")
-set(ENABLE_HIP Off CACHE BOOL "")
-set(ENABLE_SYCL Off CACHE BOOL "")
-set(ENABLE_NUMA Off CACHE BOOL "")
-set(ENABLE_OPENMP Off CACHE BOOL "")
-set(ENABLE_COPY_HEADERS Off CACHE BOOL "")
-set(ENABLE_GMOCK On CACHE BOOL "")
```

Comment:
I think you should restore this line - this changes BLT's default value for our builds (but can still be manually changed by the user, and controlled independently by the new `UMPIRE_ENABLE_GMOCK` dependent option)

Reply:
Ok, that makes sense. Hoping to resolve this today!